mindmeld/models/query_features.py

```diff
@@ -529,7 +529,7 @@ def _extractor(query, resources):
         for entity in system_entities:
             for i in entity.token_span:
                 for j in start_positions:
-                    if 0 < i - j < len(feat_seq):
+                    if 0 <= i - j < len(feat_seq):
```

Comment:
Should a similar logic apply to the other extreme as well?
```suggestion
                    if 0 <= i - j <= len(feat_seq):
```

Reply:
No, since the access pattern is `feat_seq[i - j]`, `i-j < len(feat_seq)`